app/components/snippet-list-form.js

```diff
@@ -4,6 +4,7 @@ import { action } from '@ember/object';
 import { restartableTask, task, timeout } from 'ember-concurrency';
 import { tracked } from '@glimmer/tracking';
 import { isBlank } from '../utils/strings';
+import { saveCollatedImportedResources } from '../utils/imported-resources';
```

Comment:
seems like this util got (re)moved? it's throwing an error that it can't find the import, and neither can I by grepping in the sourcecode

Reply:
Damn. Fixed that now, sorry about that.